### PR TITLE
Simplified the config. Aim is to get the actual secrets in credstash …

### PR DESCRIPTION
**Pull request by minglis**
…to be env specific, and not the random collection of things we have at the moment.

Secret definition also includes env specific things such as URLs / Queue prefixes / URLs for providers and so on.

Matches:
https://github.com/alphagov/notifications-admin/pull/928

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -5,27 +5,61 @@
...
+    ADMIN_CLIENT_USER_NAME = 'notify-admin'
+    AWS_REGION = 'eu-west-1'
+    INVITATION_EXPIRATION_DAYS = 2
+    INVITATION_EMAIL_FROM = 'no-reply'
     NOTIFY_APP_NAME = 'api'
     NOTIFY_LOG_PATH = '/var/log/notify/application.log'
     # Notification Queue names are a combination of a prefix plus a name
-    NOTIFICATION_QUEUE_PREFIX = os.environ['NOTIFICATION_QUEUE_PREFIX']
-    SECRET_KEY = os.environ['SECRET_KEY']
+    NOTIFICATION_QUEUE_PREFIX = 'development'
```
Comment (leohemsted) [inline config.py]: this is overriding line 41's env var

File: config.py [hunk 2]
```diff
@@ -5,27 +5,61 @@
...
+    ###########################
+    # Default config values ###
+    ###########################
+
+    DEBUG = False
+    NOTIFY_ENVIRONMENT = 'development'
+    ADMIN_CLIENT_USER_NAME = 'notify-admin'
+    AWS_REGION = 'eu-west-1'
+    INVITATION_EXPIRATION_DAYS = 2
+    INVITATION_EMAIL_FROM = 'no-reply'
```
Comment (leohemsted) [inline config.py]: this isn't used anywhere

File: config.py [hunk 3]
```diff
@@ -5,27 +5,61 @@
...
     # Notification Queue names are a combination of a prefix plus a name
-    NOTIFICATION_QUEUE_PREFIX = os.environ['NOTIFICATION_QUEUE_PREFIX']
-    SECRET_KEY = os.environ['SECRET_KEY']
+    NOTIFICATION_QUEUE_PREFIX = 'development'
     SQLALCHEMY_COMMIT_ON_TEARDOWN = False
-    SQLALCHEMY_DATABASE_URI = os.environ['SQLALCHEMY_DATABASE_URI']
     SQLALCHEMY_RECORD_QUERIES = True
     SQLALCHEMY_TRACK_MODIFICATIONS = True
-    NOTIFY_EMAIL_DOMAIN = os.environ['NOTIFY_EMAIL_DOMAIN']
+    NOTIFY_EMAIL_DOMAIN = 'notify.tools'
```
Comment (leohemsted) [inline config.py]: this should be defined in `Development` rather than in base config i think

File: config.py [hunk 4]
```diff
@@ -96,26 +130,26 @@ class Config(object):
...
     ]
-    API_HOST_NAME = os.environ['API_HOST_NAME']
-    MMG_API_KEY = os.environ['MMG_API_KEY']
-    FIRETEXT_API_KEY = os.getenv("FIRETEXT_API_KEY")
-    LOADTESTING_NUMBER = os.getenv('LOADTESTING_NUMBER')
-    LOADTESTING_API_KEY = os.getenv("LOADTESTING_API_KEY")
-    CSV_UPLOAD_BUCKET_NAME = os.getenv("CSV_UPLOAD_BUCKET_NAME")
+    API_HOST_NAME = "http://localhost:6011"
+
+    CSV_UPLOAD_BUCKET_NAME = 'development-notifications-csv-upload'
```
Comment (leohemsted) [inline config.py]: same with this being in Dev - i just think it's a bit nicer that if we're defining a new environment it's immediately obvious that we aren't using the dev bucket

Comment (minglis) [inline config.py]: Moved to dev config. Each env now defines own bucket with no default.

File: config.py (hunk 3 above)
Comment (minglis) [inline config.py]: Done

File: config.py (hunk 2 above)
Comment (minglis) [inline config.py]: Removed

File: config.py (hunk 1 above)
Comment (minglis) [inline config.py]: Deleted line 41

**Comment by leohemsted** [comment]
👍

**Pull request merged**